src/gwt/src/org/rstudio/studio/client/workbench/views/environment/EnvironmentTab.java

```diff
@@ -68,7 +69,7 @@ public EnvironmentTab(final Shim shim,
                          Commands commands,
                          Session session)
    {
-      super("Environment", shim);
+      super(constants_.environmentCapitalized(), shim);
```

Comment:
There may be non-obvious side effects of changing this. I need to do further research. Might be best to leave this one string unchanged for now until I can look into it. See similar feedback in this PR: #10196

> I need to do some research on this. I suspect this may prove problematic without some additional work and this is very sensitive / fragile code with some non-obvious assumptions.

Reply:
Ahh that one eh, I was wondering what would happen here. I was wondering about a possible interaction with the 
`eventBus_.fireEvent(new ActivatePaneEvent(constants_.environmentCapitalized()));` present in the EnvironmentPresenter.

Should that be changed as well?

Reply:
For this PR, just put back to original hardcoded ("Environment") in both EnvironmentPane and EnvironmentTab. There are potentially some other problems that I need to research before we change this (and a bunch of similar things for other tabs and panes).